Apps/APN-UIKit/APN UIKit/Router/DashboardRouter.swift

```diff
@@ -28,9 +28,9 @@ class DashboardRouter: DashboardRouting {
 
     func routeToSettings(_ withInfo: [String: String]? = nil) {
         let viewController = SettingsViewController.newInstance()
-        if let siteId = withInfo?["site_id"], let apiKey = withInfo?["api_key"] {
+        if let siteId = withInfo?["site_id"], let cdpWriteKey = withInfo?["write_key"] {
```

Comment:
> Are there consequences to us changing the deep link URL format from using api_key to write_key 

You are correct

> Will this slow down our QA testing?
I do not believe this will slow down QA in any way.

> I wonder if at least it's worth mentioning this change to the team to notify everyone?
I will let the team know about this change.